instrumentation/apache-camel-2.20/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/apachecamel/decorators/Http4SpanDecorator.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.apachecamel.decorators;
+
+import javax.annotation.Nullable;
+import org.apache.camel.Endpoint;
+import org.apache.camel.Exchange;
+
+class Http4SpanDecorator extends HttpSpanDecorator {
+  @Nullable
+  @Override
+  protected String getHttpUrl(Exchange exchange, Endpoint endpoint) {
+    Object url = exchange.getIn().getHeader(Exchange.HTTP_URL);
+    if (url instanceof String) {
+      return ((String) url).replace("http4", "http");
```

Comment:
In https://camel.apache.org/components/2.x/http4-component.html they also mention urls starting with `https4://`

Reply:
Nice catch! Thanks!
fixed